lib/item_marketplace/presenters/items_index_presenter.rb

```diff
@@ -5,4 +5,8 @@ class ItemsIndexPresenter
   def initialize(items)
     @items = items
   end
+
+  def available_items
+    @items.select {|item| item.available == true}
```

Comment:
In ActiveRecord, boolean attributes give you the method <attribute>?, so this could more concisely be written as `@items.select(&:available?)`